ext/InverseFunctionsCompositionsBaseExt.jl

```diff
@@ -0,0 +1,10 @@
+module InverseFunctionsCompositionsBaseExt
```

Comment:
```suggestion
module CompositionsBaseInverseFunctionsExt
```
I think package first is the more standard convention?

Reply:
Totally!